### ISSUE TITLE
Can't finish the Roblox Wine Install

### ISSUE THREAD

**Issue by yosua-wisnu** [roblox-linux-wrapper/roblox-linux-wrapper#138]
Tried this on 2 different distro based on Ubuntu 14.04 but always got the same result. What did I do wrong? Thanks
```
me@Freya:~$ rlw
Roblox Linux Wrapper 2015.06.16
wine path set to /usr/bin/wine
wineboot path set to /usr/bin/wineboot
wineserver path set to /usr/bin/wineserver
 > Calling wineboot...
wine: created the configuration directory '/home/me/.local/share/wineprefixes/roblox-wine'
err:ole:marshal_object couldn't get IPSFactory buffer for interface {00000131-0000-0000-c000-000000000046}
err:ole:marshal_object couldn't get IPSFactory buffer for interface {6d5140c1-7436-11ce-8034-00aa006009fa}
err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hres=0x80040155
err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, 80040155
err:ole:get_local_server_stream Failed: 80040155
err:ole:marshal_object couldn't get IPSFactory buffer for interface {00000131-0000-0000-c000-000000000046}
err:ole:marshal_object couldn't get IPSFactory buffer for interface {6d5140c1-7436-11ce-8034-00aa006009fa}
err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hres=0x80040155
err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, 80040155
err:ole:get_local_server_stream Failed: 80040155
fixme:ntdll:NtLockFile I/O completion on lock not implemented yet
err:mscoree:LoadLibraryShim error reading registry key for installroot
err:mscoree:LoadLibraryShim error reading registry key for installroot
err:mscoree:LoadLibraryShim error reading registry key for installroot
err:mscoree:LoadLibraryShim error reading registry key for installroot
fixme:ntdll:NtLockFile I/O completion on lock not implemented yet
fixme:iphlpapi:NotifyIpInterfaceChange (family 0, callback 0x69e6d901, context 0x713de0, init_notify 0, handle 0xfde7a4): stub
fixme:iphlpapi:CancelMibChangeNotify2 (handle (nil)): stub
wine: configuration in '/home/me/.local/share/wineprefixes/roblox-wine' has been updated.
> rwinetricks: calling winetricks with arguments "ddr=gdi sound=alsa "
> rwinetricks: winetricks path set to /usr/lib/roblox-linux-wrapper/winetricks
Executing w_do_call ddr=gdi
Executing load_ddr gdi
Setting Direct3D/DirectDrawRenderer to gdi
Executing winetricks_early_wine regedit C:\windows\Temp\_ddr=gdi\set-wined3d.reg
Executing w_do_call sound=alsa
Executing load_sound alsa
Setting sound driver to alsa
Executing winetricks_early_wine regedit C:\windows\Temp\_sound=alsa\set-sound.reg
--2015-07-22 22:02:42--  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Resolving download.cdn.mozilla.net (download.cdn.mozilla.net)... 118.98.37.25, 118.98.37.8
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 22:04:57--  (try: 2)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 22:07:06--  (try: 3)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 22:09:17--  (try: 4)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... 
```

**Comment by alfonsojon** [comment]
It looks like download.cdn.mozilla.net is redirecting so some other IP. Have you tried restarting your computer? If this does not work, try [changing your DNS servers to OpenDNS](https://www.opendns.com/) or another public DNS service.

**Comment by yosua-wisnu** [comment]
Thanks for the reply, I applied OpenDNS and restarted my laptop, still stuck at the same place.

**Comment by yosua-wisnu** [comment]
Third try plus a little bit more patience, it won't run because of the firefox tho.

me@Freya:~$ rlw
Roblox Linux Wrapper 2015.06.16
No Wine path has been saved yet, starting winechooser...
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
Wine path set to: $(which wine)
Saving Wine choice to ~/.rlw/wine_choice
wine path set to /usr/bin/wine
wineboot path set to /usr/bin/wineboot
wineserver path set to /usr/bin/wineserver
wine path set to /usr/bin/wine
wineboot path set to /usr/bin/wineboot
wineserver path set to /usr/bin/wineserver
 > Calling wineboot...
wine: created the configuration directory '/home/me/.local/share/wineprefixes/roblox-wine'
err:ole:marshal_object couldn't get IPSFactory buffer for interface {00000131-0000-0000-c000-000000000046}
err:ole:marshal_object couldn't get IPSFactory buffer for interface {6d5140c1-7436-11ce-8034-00aa006009fa}
err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hres=0x80040155
err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, 80040155
err:ole:get_local_server_stream Failed: 80040155
err:ole:marshal_object couldn't get IPSFactory buffer for interface {00000131-0000-0000-c000-000000000046}
err:ole:marshal_object couldn't get IPSFactory buffer for interface {6d5140c1-7436-11ce-8034-00aa006009fa}
err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hres=0x80040155
err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, 80040155
err:ole:get_local_server_stream Failed: 80040155
fixme:ntdll:NtLockFile I/O completion on lock not implemented yet
err:mscoree:LoadLibraryShim error reading registry key for installroot
err:mscoree:LoadLibraryShim error reading registry key for installroot
err:mscoree:LoadLibraryShim error reading registry key for installroot
err:mscoree:LoadLibraryShim error reading registry key for installroot
fixme:ntdll:NtLockFile I/O completion on lock not implemented yet
fixme:iphlpapi:NotifyIpInterfaceChange (family 0, callback 0x69e6d901, context 0x713de0, init_notify 0, handle 0xfde7a4): stub
fixme:iphlpapi:CancelMibChangeNotify2 (handle (nil)): stub
wine: configuration in '/home/me/.local/share/wineprefixes/roblox-wine' has been updated.
> rwinetricks: calling winetricks with arguments "ddr=gdi sound=alsa "
> rwinetricks: winetricks path set to /usr/lib/roblox-linux-wrapper/winetricks
Executing w_do_call ddr=gdi
Executing load_ddr gdi
Setting Direct3D/DirectDrawRenderer to gdi
Executing winetricks_early_wine regedit C:\windows\Temp\_ddr=gdi\set-wined3d.reg
Executing w_do_call sound=alsa
Executing load_sound alsa
Setting sound driver to alsa
Executing winetricks_early_wine regedit C:\windows\Temp\_sound=alsa\set-sound.reg
--2015-07-22 23:21:08--  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Resolving download.cdn.mozilla.net (download.cdn.mozilla.net)... 118.98.37.25, 118.98.37.8
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:23:17--  (try: 2)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:25:27--  (try: 3)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:27:38--  (try: 4)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:29:50--  (try: 5)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:32:03--  (try: 6)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:34:17--  (try: 7)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:36:32--  (try: 8)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:38:48--  (try: 9)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:41:05--  (try:10)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:43:23--  (try:11)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:45:40--  (try:12)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:47:58--  (try:13)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:50:15--  (try:14)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:52:33--  (try:15)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:54:50--  (try:16)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:57:08--  (try:17)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-22 23:59:25--  (try:18)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-23 00:01:42--  (try:19)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Retrying.

--2015-07-23 00:03:59--  (try:20)  http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.25|:80... failed: Connection refused.
Connecting to download.cdn.mozilla.net (download.cdn.mozilla.net)|118.98.37.8|:80... failed: Connection timed out.
Giving up.

chmod: cannot access ‘/tmp/Firefox-Setup/*.exe’: No such file or directory
wine: cannot find '/tmp/Firefox-Setup/*.exe'
> rwget: calling wget with arguments "http://roblox.com/install/setup.ashx -O /tmp/RobloxPlayerLauncher.exe "
> rwine: calling wine with arguments "/tmp/RobloxPlayerLauncher.exe "
wine: cannot find L"C:\\windows\\system32\\winemenubuilder.exe"
err:wineboot:ProcessRunKeys Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (2)
fixme:ole:RemUnknown_QueryInterface No interface for iid {00000019-0000-0000-c000-000000000046}
fixme:mstask:MSTASK_ITaskScheduler_Delete 0x147b40, L"RobloxGameUpdater": stub
fixme:mstask:MSTASK_ITaskScheduler_Delete 0x147b40, L"RobloxGameUpdater": stub
fixme:mstask:MSTASK_ITaskScheduler_Delete 0x13eb68, L"RobloxGameUpdater": stub
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
> rwineserver: calling wineserver with arguments "--wait "
wine: cannot find L"C:\\windows\\system32\\winemenubuilder.exe"
wine: cannot find L"C:\\windows\\system32\\winemenubuilder.exe"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
fixme:wininet:create_cache_entry INTERNET_FLAG_NEED_FILE is not supported correctly
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
wine: cannot find L"C:\\windows\\system32\\winemenubuilder.exe"
wine: cannot find L"C:\\windows\\system32\\winemenubuilder.exe"
wine: cannot find L"C:\\windows\\system32\\winemenubuilder.exe"
wine: cannot find L"C:\\windows\\system32\\winemenubuilder.exe"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
fixme:wininet:IsDomainLegalCookieDomainW (L"roblox.com", L"www.roblox.com") semi-stub
fixme:wininet:IsDomainLegalCookieDomainW (L"roblox.com", L"www.roblox.com") semi-stub
fixme:wininet:IsDomainLegalCookieDomainW (L"roblox.com", L"www.roblox.com") semi-stub
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
fixme:wininet:create_cache_entry INTERNET_FLAG_NEED_FILE is not supported correctly
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
fixme:wininet:create_cache_entry INTERNET_FLAG_NEED_FILE is not supported correctly
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
err:wininet:HTTP_ParseDateAsAsctime unexpected weekday L"-1"
err:wininet:HTTP_ParseDate unexpected date format L"-1"
> rwine: calling wine with arguments "regsvr32 /i /home/me/.local/share/wineprefixes/roblox-wine/drive_c/users/me/Local Settings/Application Data/RobloxVersions/version-4993687f79834cd9/RobloxProxy.dll "
fixme:ole:RemUnknown_QueryInterface No interface for iid {00000019-0000-0000-c000-000000000046}
fixme:ole:RemUnknown_QueryInterface No interface for iid {00000019-0000-0000-c000-000000000046}
regsvr32: Successfully registered DLL '/home/me/.local/share/wineprefixes/roblox-wine/drive_c/users/me/Local Settings/Application Data/RobloxVersions/version-4993687f79834cd9/RobloxProxy.dll'
fixme:msvcrt:__clean_type_info_names_internal (0x1001c398) stub
regsvr32: Successfully installed DLL '/home/me/.local/share/wineprefixes/roblox-wine/drive_c/users/me/Local Settings/Application Data/RobloxVersions/version-4993687f79834cd9/RobloxProxy.dll'
fixme:msvcrt:__clean_type_info_names_internal (0x1001c398) stub
> rwineserver: calling wineserver with arguments "--wait "
> rwine: calling wine with arguments "/home/me/.local/share/wineprefixes/roblox-wine/drive_c/Program Files/Mozilla Firefox/firefox.exe http://www.roblox.com/Games.aspx "
wine: cannot find '/home/me/.local/share/wineprefixes/roblox-wine/drive_c/Program Files/Mozilla Firefox/firefox.exe'
me@Freya:~$

**Comment by alfonsojon** [comment]
Apply OpenDNS on your router, if you can.

**Comment by GLolol** [comment]
The CDN is probably resolving to different addresses based on your location. If you set your DNS on your computer, you don't need to apply it again on your router...

**Comment by alfonsojon** [comment]
@GLolol I was imagining the ISP interfering somehow.

**Comment by GLolol** [comment]
Setting it via the router wouldn't make a difference if OpenDNS is entirely blocked.

**Comment by alfonsojon** [comment]
Ah. Well anyways, @yosua-wisnu, try running the following command: 

```
wget http://ftp.mozilla.org/pub/mozilla.org/firefox/releases/latest/win32/en-US/Firefox%20Setup%2039.0.exe -O /tmp/Firefox.exe && WINEPREFIX=~/.local/share/wineprefixes/roblox-wine wine /tmp/Firefox.exe
```
After it finishes installing, it should hopefully work.

**Comment by yosua-wisnu** [comment]
Yeah my ISP is a restricting one, they even blocked reddit.

The first part of that command didnt work, but i understand what you meant. I downloaded,  move and renamed  the setup file to ./temp/Firefox.exe. the second part of the command works very well.

I tried 3 games, The first one is "Work at the pizza place" it shows  "Game Shutdown" and roblox crash. The second one is "Animation Lab 2.0" roblox crash on game loading screen. The third one is "Dragon Rage" after loading screen it shows "Connection Lost" 

Sorry for the long line 
https://gist.githubusercontent.com/yosua-wisnu/77637613a07c86b853ed/raw/07676f60ed27436e0b995dbc69fb5081c8b6fae5/rlw-log-2
